Samples/IndexImplementation/Program.cs

```diff
@@ -334,22 +334,23 @@ static void AllocND<T>(Accelerator accelerator, Func<MyIndex4, MyIndex4, T> valu
                     }
                 }
             }
+
```

Comment:
QUESTION: The code formatter appears to add liberal amounts of new lines to separate code blocks. It particularly likes to add a new line after a closing brace. Are we able to disable this?

Reply:
this should be fixed now 🚀